packages/fxa-settings/src/pages/CannotCreateAccount/en.ftl

```diff
@@ -0,0 +1,7 @@
+## Cannot Create Account page
+## Users are redirected to this page if they attempt to create an account that does not meet age requirements.
+
+cannot-create-account = Cannot create account
```

Comment:
What's the context of this string? Title/header? Error? If you could add that context as an ID would be great.

Reply:
It's a header, I'll change this to `cannot-create-account-header`. 👍